model/edition.py

```diff
@@ -707,7 +721,10 @@ def choose_cover(self):
             # Identifier. Try to find a thumbnail the same way we'd
             # look for a cover.
             for distance in (0, 5):
-                best_thumbnail, thumbnails = self.best_cover_within_distance(distance, rel=LinkRelations.THUMBNAIL_IMAGE)
+                best_thumbnail, thumbnails = self.best_cover_within_distance(
+                    distance=distance, policy=policy,
+                    rel=LinkRelations.THUMBNAIL_IMAGE,
+                )
```

Comment:
`best_cover_within_distance` is weird because we're using part of the incoming policy but not all of it -- we're overriding the `levels` variable and going one level deeper every time we fail to find a cover. That's because we're only going to choose one cover. A cover associated with an Identifier that's closer to the LicensePool is better than a cover associated with an Identifier that's further away, even if the further-away cover is bigger or has a better aspect ratio, because there's less of a risk that it's a cover for a totally different book.

Now that I think about it, the situation is similar with descriptions, but we don't actually pick descriptions this way. I added https://jira.nypl.org/browse/SIMPLY-1696 to address that.